### ISSUE TITLE
Unchecked "Enable Lobby Chat" In Valorant But Can Still See In-Game Chat

### ISSUE THREAD

**Issue by danblock97** [molenzwiebel/Deceive#230]
I know the option specifies **Lobby** but when in game, in the chat I see "Error: Chat room not connected" which leads me to believe that the intention might also be in game chat too, if this is the case, I can still see people chatting. If this isn't the case then I guess working as intended but thought I'd rase just incase

![image](https://github.com/molenzwiebel/Deceive/assets/109729853/4ce6bff8-6aaa-49ab-938a-f2042d4031c4)

**Comment by molenzwiebel** [comment]
Yeah, this is a remnant from when Deceive was League-only. League's in-game chat works using a different chat system, so it'd still work. VALORANT uses the same chat system for lobby and ingame, so it breaks. Interesting that it manifests in this way.

Should probably look at renaming this option (or removing it entirely; not sure if many people use it at all).

**Comment by danblock97** [comment]
@molenzwiebel Ahh I see all good, yeah I've used it on League, never really on Valorant just mainly to appear offline in Valorant.
